HaxeManual/12-target-details.tex

```diff
@@ -253,6 +253,10 @@ \subsection{ECMAScript 6 class generation}
 \subsection{Using external JavaScript libraries}
 \label{target-javascript-external-libraries}
 
+\since{4.0.0}
+
+\emph{Deprecated:} The jQuery and SWFObject externs have been removed from the standard library in Haxe 4. For compatibility purposes they are available in the \href{https://github.com/HaxeFoundation/hx3compat/}{hx3compat} library.
```

Comment:
Missing `\type{}`?

Reply:
(per Slack chat, these refer to the library names, no reason to `\type`)